planning/motion_velocity_planner/autoware_motion_velocity_out_of_lane_module/src/calculate_slowdown_points.cpp

```diff
@@ -127,4 +127,26 @@ std::optional<geometry_msgs::msg::Pose> calculate_slowdown_point(
   }
   return slowdown_point;
 }
+
+void calculate_min_stop_and_slowdown_distances(
+  out_of_lane::EgoData & ego_data, const PlannerData & planner_data,
+  const std::optional<geometry_msgs::msg::Pose> & previous_slowdown_pose,
+  const double slow_velocity)
+{
+  ego_data.min_stop_distance = planner_data.calculate_min_deceleration_distance(0.0).value_or(0.0);
+  ego_data.min_slowdown_distance =
```

Comment:
It seems that both `min_stop_distance` and `min_slowdown_distance` are only relevant within this function, so I wonder if they need to be part of `EgoData` struct.

Additionally it doesn't seem that `min_slow_distance` has any significance, its just being computed but not used anywhere.